versions.props

```diff
@@ -73,6 +73,7 @@ org.hdrhistogram:HdrHistogram = 2.1.10
 org.hibernate:hibernate-validator = 5.1.3.Final
 org.immutables:* = 2.7.3
 org.javassist:javassist = 3.18.2-GA
+org.jmock:jmock = 2.8.2
```

Comment:
Remove from gradle/versions.gradle and update timestamp-impl?

Reply:
Will do